pyiron_atomistics/__init__.py

```diff
@@ -3,25 +3,41 @@
 
 from pyiron_atomistics.project import Project
 from pyiron_atomistics.toolkit import AtomisticsTools
-from pyiron_atomistics.atomistics.structure.atoms import ase_to_pyiron, pyiron_to_ase, Atoms
+from pyiron_atomistics.atomistics.structure.atoms import (
+    ase_to_pyiron,
+    pyiron_to_ase,
+    Atoms,
+)
 from pyiron_base import Notebook, install_dialog, JOB_CLASS_DICT
 
 from pyiron_base import Project as ProjectBase
-ProjectBase.register_tools('atomistics', AtomisticsTools)
+
+ProjectBase.register_tools("atomistics", AtomisticsTools)
 
 # To maintain backwards compatibility until we deprecate the old structure creation functions:
-from pyiron_atomistics.atomistics.structure.factory import StructureFactory as _StructureFactory
+from pyiron_atomistics.atomistics.structure.factory import (
+    StructureFactory as _StructureFactory,
+)
+
 create_surface = _StructureFactory.surface
 create_ase_bulk = _StructureFactory().ase.bulk
 create_structure = _StructureFactory.crystal
 
 # Make classes available for new pyiron version
 JOB_CLASS_DICT["ART"] = "pyiron_atomistics.interactive.activation_relaxation_technique"
 JOB_CLASS_DICT["AtomisticExampleJob"] = "pyiron_atomistics.testing.randomatomistic"
-JOB_CLASS_DICT["ConvEncutParallel"] = "pyiron_atomistics.dft.master.convergence_encut_parallel"
-JOB_CLASS_DICT["ConvEncutSerial"] = "pyiron_atomistics.dft.master.convergence_encut_serial"
-JOB_CLASS_DICT["ConvergenceVolume"] = "pyiron_atomistics.atomistics.master.convergence_volume"
-JOB_CLASS_DICT["ConvKpointParallel"] = "pyiron_atomistics.dft.master.convergence_kpoint_parallel"
+JOB_CLASS_DICT[
```

Comment:
Maybe it is more elegant to define a dictionary and then iterate over the dictionary to copy the entries to the `JOB_CLASS_DICT`.